README.md

```diff
@@ -1,4 +1,7 @@
-#Homework №3
+# Lesson_05 Classwork - Basic calculator
```

Comment:
Some classwork from future :)

Reply:
And it’s not because I invented a time machine :-D